edge/pkg/devicetwin/dtcontext/dtcontext_test.go

```diff
@@ -27,6 +26,7 @@ import (
 	"github.com/kubeedge/kubeedge/edge/pkg/common/modules"
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtcommon"
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dttype"
+	devicetest "github.com/kubeedge/kubeedge/edge/test/device_test"
```

Comment:
We'd better not import the edge/test package from edge/pkg, the edge/test package is used for integration test, mabe we can put the code in devicetwin/testutil directory

Reply:
OK

Reply:
I have move the code to devicetwin/testutil